### PR TITLE
add runlike

### PR DESCRIPTION
**Pull request by Moshe-Immerman**

### CODE REVIEW

**Comment by Moshe-Immerman** [comment]
runlike was already listed under wrappers - however docker-replay which is the alternative is under builders - arguments can be made for both as a wrapper and a builder but whichever way it goes, they should both be in the same section.

File: README.md [hunk 1]
```diff
@@ -335,6 +335,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 * [MicroBadger][microbadger] - Analyze the contents of images and add metadata labels
 * [packer](https://www.packer.io/docs/builders/docker.html) - Hashicorp tool to build machine images including docker image integrated with configuration management tools like chef, puppet, ansible
 * [portainer](https://github.com/duedil-ltd/portainer) - Apache Mesos framework for building Docker images by [@duedil-ltd](https://github.com/duedil-ltd)
+* [runlike](https://github.com/lavie/runlike) - Generate `docker run`command and options from running containers by [@lavie](https://github.com/lavie)
```
Comment (devhkr) [inline README.md]: According to the project description:

>This is very much a work in progress. Many docker run options aren't yet supported, but the most commonly used ones are. Feel free to send pull requests if you add any or if you happen to fix any (of the many) bugs this package undoubtedly has.
>
>Probably shouldn't use this in production yet. If you do, double check that it's actually running what you want it to run.

Please add in the description that it's a WIP, or putting a symbol after the link is better like

```
[runlike](https://github.com/lavie/runlike) :face_with_head_bandage: - Generate
```

Comment (devhkr) [reply README.md]: :nerd_face: symbol for CLI tools is missing, too.

```
[runlike](https://github.com/lavie/runlike) :face_with_head_bandage: :nerd_face: - Generate
```

**Comment by devhkr** [comment]
@veggiemonk I commented on the issue #446 on how about using the 🔨 as for projects work in progress and for beta status.

**Comment by agebhar1** [comment]
... or :hammer_and_wrench: or :construction: (https://github.com/veggiemonk/awesome-docker/issues/446#issuecomment-328097279)

**Comment by veggiemonk** [comment]
I like this one 🚧

**Comment by devhkr** [comment]
Please add :construction: so we can merge the PR.

**Pull request merged**